stdlib/logging/__init__.pyi

```diff
@@ -301,10 +303,10 @@ class Formatter:
 
```

Comment:
This class has `_style: PercentStyle`, but I can't comment on that line specifically because github. Maybe it should be `PercentStyle | StrFormatStyle | StringTemplateStyle`?

Reply:
Surely you refer to
https://github.com/python/typeshed/blob/ee1a00c9a001694c6e74b4c9cb51558ee756142b/stdlib/logging/__init__.pyi#L297
defined by
https://github.com/python/typeshed/blob/ee1a00c9a001694c6e74b4c9cb51558ee756142b/stdlib/logging/__init__.pyi#L799-L810

I supposed the undocumented `_style` parameter referred to undocumented classes unrelated to the value of these `style` parameters documented as literal strings.

> The style parameter can be one of ‘%’, ‘{‘ or ‘$’ and determines how the format string will be merged with its data: using one of `%`-formatting, `str.format()` or `string.Template`.

Is that not the case?

Reply:
I was asking to update `_style`. We can do it in this PR or in a separate PR or not at all, however you prefer.

Reply:
I was about to fix `_style`, then noticed the class inheritance:
https://github.com/python/typeshed/blob/ee1a00c9a001694c6e74b4c9cb51558ee756142b/stdlib/logging/__init__.pyi#L812-L817
Though counterintuitive, I think the hint was chosen deliberately.
It's technically correct.

Reply:
You're right. Let's leave it as is.